shortfin/python/shortfin_apps/sd/server.py

```diff
@@ -130,9 +130,10 @@ def configure_service(args, sysman, model_config, flagfile, tuning_spec):
     for idx, tok_name in enumerate(args.tokenizers):
         subfolder = f"tokenizer_{idx + 1}" if idx > 0 else "tokenizer"
         tokenizers.append(Tokenizer.from_pretrained(tok_name, subfolder))
-
     model_params = ModelParams.load_json(model_config)
     vmfbs, params = get_modules(args, model_config, flagfile, tuning_spec)
+    logger.info(f"vmfbs: {vmfbs}")
+    logger.info(f"params: {params}")
```

Comment:
If these logs are useful to keep, want to add more context?

For example:
```
Service configured!
  Program files: foo.vmfb, bar.vmfb
  Parameters: baz.irpa
```

Reply:
I'll remove them. Debug prints.